src/Components/Admins/index.js

```diff
@@ -72,7 +74,10 @@ const Admins = () => {
         text="Are you sure you want to delete the admin selected?"
       ></Modal>
       <ErrorMessage show={showErrorMessage} close={closeError} text={errorMessageText} />
-      <h2 className={styles.header}>Admins</h2>
+      <div className={styles.titleAndButton}>
+        <h3>Admin</h3>
+        <Button onClick={() => addAdmin()} value="Admin" />
```

Comment:
If you don't need to send any extra parameter through this place you can do this:
```suggestion
        <Button onClick={addAdmin} value="Admin" />
```